app/controllers/shows_controller.rb

```diff
@@ -0,0 +1,4 @@
+class ShowsController < ApplicationController
```

Comment:
Probably a personal thing, but when I look at the naming of the controller I get a bit confused about what resource 'shows' refers to - would titles/pages/sections/views be more explicit? WDYT? Or will it eventually be a page for displaying movies?

Reply:
In such cases, people also call the controller like `WelcomeController`, `HomeController`, `PagesController` etc. https://stackoverflow.com/questions/349743/welcome-home-page-in-ruby-on-rails-best-practice